builder/builder-next/adapters/snapshot/snapshot.go

```diff
@@ -242,7 +243,7 @@ func (s *snapshotter) Stat(ctx context.Context, key string) (snapshots.Info, err
 	if err := s.db.View(func(tx *bolt.Tx) error {
 		b := tx.Bucket([]byte(id))
 		if b == nil && l == nil {
-			return errors.Errorf("snapshot %s not found", id) // TODO: typed
+			return errdefs.ErrNotFound
```

Comment:
I think these can be wrapped if the additional information is useful, e.g.;

```go
return errors.Wrapf(errdefs.ErrNotFound, "snapshot %s", id)
```

That's similar to how containerd itself uses these; https://github.com/moby/moby/blob/d5d5f258dfc95c46ed1e62953a754b7cf3edecd3/vendor/github.com/containerd/containerd/container.go#L201

Reply:
@crazy-max ^